pom.xml

```diff
@@ -78,6 +78,11 @@
                 <artifactId>servlet-api</artifactId>
                 <version>2.5</version>
             </dependency>
+            <dependency>
+                <groupId>javax.servlet</groupId>
+                <artifactId>javax.servlet-api</artifactId>
+                <version>4.0.1</version>
+            </dependency>

```

Comment:
You should probably use our own version of the spec api:

`<groupId>org.jboss.spec.javax.servlet</groupId>`
`<artifactId>jboss-servlet-api_4.0_spec</artifactId>`